content/en/core/guides/using-windows.md

```diff
@@ -110,6 +110,10 @@ $: cd ~/medic/cht-core
 $: code .
 ```
 
-## Problems?
+## Default port and credentials for cht-core web
+The default launch port for cht-core is 5988, which can be changed in cht-core/api/server.js.
```

Comment:
I think the recommended approach would be to provide an environment variable rather than editing the code - that way the change won't be lost if you upgrade.

```suggestion
The default launch port for cht-core is 5988, which can be changed by providing the environment variable at runtime `API_PORT`, for example: `API_PORT=6000 node server.js`.
```